hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -216,6 +216,7 @@ public class HoodieWriteConfig extends HoodieConfig {
   public static final ConfigProperty<String> AVRO_SCHEMA_STRING = ConfigProperty
       .key("hoodie.avro.schema")
       .noDefaultValue()
+      .markAdvanced()
```

Comment:
I think this may still be used for the basic schema evolution.  Let's not make it advanced.